openjpa-kernel/src/main/java/org/apache/openjpa/event/TCPRemoteCommitProvider.java

```diff
@@ -145,16 +145,16 @@ public int getRecoveryTimeMillis() {
      * The maximum number of sockets that this provider can
      * simetaneously open to each peer in the cluster.
      */
-    public void setMaxActive(int maxActive) {
```

Comment:
I would _also_ keep the old method but log a warning which states that it might be removed and people shall use `setMaxTotal(int)` instead. Probably also deprecate it.